src/site/content/en/learn/css/transitions/index.md

```diff
@@ -0,0 +1,251 @@
+
+{% Aside 'key-term' %}
+_Interpolation_ is the process of creating "in-between" steps that smoothly transition from one state to another.
+{% endAside %}
+
+{% Codepen {
+  user: 'web-dot-dev',
+  id: 'zYzNrJV',
+  height: 320
+} %}
+
+## Transition properties
+
+To use transitions in CSS, you can use the various transition properties or the `transition` shorthand property. 
+
+### transition-property
+
+The [`transition-property`](https://developer.mozilla.org/docs/Web/CSS/transition-property) property specifies which style(s) to transition.
+
+```css
+.my-element {
+  transition-property: background-color;
+}
+```
+
+The `transition-property` accepts one or more CSS property names in a comma-separated list.
+
+Optionally, you may use `transition-property: all` to indicate that every property should transition.
+
+{% Codepen {
+  user: 'web-dot-dev',
+  id: 'VwWPeEj',
+  height: 400
+} %}
+
+### transition-duration
+
+The [`transition-duration`](https://developer.mozilla.org/docs/Web/CSS/transition-duration) property is used to define the length of time that a transition will take to complete.
+
+{% Codepen {
+  user: 'web-dot-dev',
+  id: 'wvegMYp',
+  height: 400
+} %}
+
+`transition-duration` accepts time units, either in seconds (`s`) or milliseconds (`ms`) and defaults to `0s`.
+
+### transition-timing-function
+
+Use the [`transition-timing-function`](https://developer.mozilla.org/docs/Web/CSS/transition-timing-function) property to vary the speed of a CSS transition over the course of the `transition-duration`.
+
+By default, CSS will transition your elements at a constant speed (`transition-timing-function: linear`). Linear transitions can end up looking somewhat artificial, though: in real life, objects have weight and can't stop and start instantly. Easing into or out of a transition can make your transitions more lively and natural.
+
+{% Codepen {
+  user: 'web-dot-dev',
+  id: 'QWgdyZx'
+} %}
+
+Our [module on CSS Animation](/learn/css/animations/#animation-timing-function) has a good overview of timing functions.
+
+You can use DevTools to experiment with different timing functions in real-time.
+
+{% Img src="image/eiKy1JcjHqPp3gaedjAQWjPJ8YK2/G06SHV4Dra1HqTAL9d6t.png", alt="Chrome DevTools visual transition timing editor.", width="264", height="359" %}
+
+### transition-delay
+
+Use the [`transition-delay`](https://developer.mozilla.org/docs/Web/CSS/transition-delay) property to specify the time at which a transition will start. If `transition-duration` is not specified, transitions will start instantly because the default value is `0s`. This property accepts a time unit, for example seconds (`s`) or milliseconds (`ms`).
+
+{% Codepen {
+  user: 'web-dot-dev',
+  id: 'jOwyWep'
+} %}
+
+This property is useful for staggering transitions, achieved by setting a longer `transition-delay` for each subsequent element in a group.
+
+{% Codepen {
```

Comment:
tiny nit on the [demo](https://codepen.io/web-dot-dev/pen/yLXgeRQ) 🙂 
all the others have a gap between items, making this one look like it's missing it

Reply:
Done, good call!